pkg/abi/abidecode.go

```diff
@@ -150,6 +161,8 @@ func decodeABILength(ctx context.Context, desc string, block []byte, offset int)
 	if offset+32 > len(block) {
 		return -1, i18n.NewError(ctx, signermsgs.MsgNotEnoughBytesABIArrayCount, desc)
 	}
+	newBlock := block[offset : offset+32]
+	fmt.Println(newBlock)
```

Comment:
Need to change this to a log line if we want to keep it in

Reply:
oh nice - was a debug log